clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -299,6 +300,35 @@ public void resume(Collection<TopicPartition> partitions) {
     }
 
     @Override
+    public Map<TopicPartition, OffsetAndTimestamp> offsetsForTimes(Map<TopicPartition, Long> timestampsToSearch) {
+        return null;
```

Comment:
I suggest throwing `UnsupportedOperationException("Not implemented yet")` so that it's obvious that it needs to be implemented still.